### PR TITLE
V2.19.0

### PR DESCRIPTION
**Pull request by misscs**
### Issues

Assortment of improvements for nightshade core that affects the storefront-static repo

* Remove utility classes on macros to make them more flexible. 
* Update footer to work for all countries. 
* Some IE fixes

I have additional PRs for strorefront static to bring it up to date with production.

### Todos

- [x] Write up testing steps for storefront static
- [x] Code/Review Testing
- [ ] Bump version

### Impacted Areas

- All headings except Upgrade section heading 
- Value props
- Footer

### Known Issues

Until updates are made to storefront static the following modules are not in parity with the live site. Most have issues at certain breakpoints. 

**All browsers**

- Upgrade callout missing heading
- Footer links and text are missing

**IE only**

- Awards
- Value props at table-ish sizes

#### On Storefront Static

These are screenshots of `master` branch without and with the changes in this branch changes. Areas that appear to be regressions from current live site will be addressed in separate commits to `storefront-static`. 

##### Chrome 

**Before/without patch (current state of master)**
ease-before-patch

**After/with patch**
ease-after-patch

##### IE

**IE Before**
ease-ie-before-patch

**IE After**
ease-ie-after-patch

To test this patch on Storefront static, create a link as outlined in directions below 👇 

#### Ensure no regressions on Casper.com Rails repo

- Change to this directory and branch. 
  - Run `nvm current`. 
  - If version is not `5.10.1`, `nvm use 5.10.1`.
- Create a link `npm link`
- Link repo to Casper Rails site `npm link @casper/nightshade-core`
- Click through PDPs, there should be any regressions

### CODE REVIEW

**Comment by misscs** [comment]
@josephco @robwierzbowski 👋 😀 

I'm working on getting storefront static master updated to its deployable state. This relies on updates to nightshade core. Can I get code review on this PR? 

Thanks so much! 

cs

**Comment by robwierzbowski** [comment]
Of course! Hi Claudina 😭😭😭🙂🙂🙂

File: src/value_props/_base.scss [hunk 1]
```diff
@@ -55,6 +60,28 @@
 
 .value-prop-heading {
   @include margin-auto;
+  @include heading-4;
   margin-bottom: ms(-1);
-  font-size: ms(0);
+
+}
+
+// Variant: Prop without an image
```
Comment (robwierzbowski) [inline src/value_props/_base.scss]: I think this is self documenting :)

File: src/value_props/_ie_styles.scss [hunk 2]
```diff
@@ -16,3 +16,9 @@
     margin: 0 7vw 0 0;
   }
 }
+
+
```
Comment (robwierzbowski) [inline src/value_props/_ie_styles.scss]: Double line break. Maybe that's normal in this repo, I see a couple below too.

**Review by robwierzbowski — COMMENTED** [review]
I'm not super familiar with this app, but all seems kosher here. DTM!

File: src/value_props/_base.scss (hunk 1 above)
Comment (misscs) [reply src/value_props/_base.scss]: I'll remove that.

File: src/value_props/_ie_styles.scss (hunk 2 above)
Comment (misscs) [reply src/value_props/_ie_styles.scss]: Naw, should follow standards.

**Comment by misscs** [comment]
@robwierzbowski Made those changes. Rebased into https://github.com/CasperSleep/nightshade-core/pull/188/commits/2447de9748f15637060aefa8284802972b6977ad. 

All that's left is to merge and bump version number. Thank you for the review!

**Pull request merged**